conda.recipe/meta.yaml

```diff
@@ -8,7 +8,6 @@ source:
   path: ../
 
 build:
-  number: 0
```

Comment:
Why remove the build number ?

Reply:
Because to be consistent, we should add one too here:

https://github.com/atoti/jdk4py/blob/8810e2faad3b7eb9fee980a99ce1ae686ea6e2a4/.github/workflows/deployment.yaml#L55

And make sure that they are kept in sync.

We could do this in a follow-up PR.

Reply:
Actually, let's do it now and try to fix atoti/atoti#279 without having to publish a new version of atoti or fighting pip/poetry/conda to ignore the pinned version of jdk4py in atoti.